gensim/models/phrases.py

```diff
@@ -232,6 +232,42 @@ def load(cls, *args, **kwargs):
         return model
 
 
+def sentence2token(phrase_class, sentence):
+    """ returns tokens after from a sentence by joining the phrases with delimeter.
```

Comment:
Please use numpy-style docstring (as almost everywhere in gensim), see
- https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard
- https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html